ui/client/containers/NussknackerInitializer.tsx

```diff
@@ -38,8 +38,8 @@ class NussknackerInitializer extends React.Component<Props, State> {
   public static HTTP_APPLICATION_CODE = 500
   public static ACCESS_TOKEN_CODE = 1024
 
-  redirectToAuthorizeUrl = () => {
-    window.location.replace(this.props.authenticationSettings.authorizeUrl)
+  redirectToAuthorizeUrl(settings) {
```

Comment:
It's tsx file so, you can add type for settigns.

Reply:
`authenticationSettings` has no type yet:
```
export type SettingsState = {
  loggedUser: Partial<User>,
  featuresSettings: $TodoType,
  authenticationSettings: $TodoType,
  analyticsSettings: $TodoType,
  processDefinitionData: ProcessDefinitionData,
  availableQueryableStates: $TodoType,
}
```
and
```
declare type $TodoType = any
```

Reply:
But you can create authenticationSettings? You know what DTO is on backend :)